src/index.js

```diff
@@ -0,0 +1,90 @@
+import './style.css';
+import checker from './checker.js';
+import addList, { addMore } from './addList.js';
+import filterTodo, { filterSingle } from './filterTodo.js';
+
+const listed = document.querySelector('.listed');
+const clearAll = document.querySelector('.clear-all');
+const clearBtn = document.querySelector('.clear-btn');
+const toDoArr = [];
+
+const getLocalStorage = () => (localStorage.getItem('todo') ? JSON.parse(localStorage.getItem('todo')) : toDoArr);
+
+export default function component(store) {
+  const listArr = store.map((i) => `
+     <li class="d-flex align-list">
+         <label for="checklist"></label>
+         <input type='checkbox' name="checklist">
+         <label for="addedlist"></label>
+         <input type="text" id="added-list" name="addedlist" class="added-section" placeholder='${i.description}' value='${i.description}' required>
+         <div class="button-flex">
+             <button type="submit" class="btn added"><i class="fa fa-ellipsis-v"></i><i class="fa fa-trash-o"></i></button>
+         </div>
+     </li>`);
+  listed.innerHTML = listArr.join('');
+  clearBtn.addEventListener('click', () => {
+    const newStore = filterTodo(store);
+    component(newStore);
+  });
+  const added = document.querySelectorAll('input[type=text][name=addedlist]');
+  const checkList = document.querySelectorAll('input[type=checkbox][name=checklist]');
+  const addedSection = document.querySelectorAll('.added-section');
+  const deleteIcon = document.querySelectorAll('.fa-trash-o');
+  const menuIcon = document.querySelectorAll('.fa-ellipsis-v');
+  const delbtn = document.querySelectorAll('.added');
+  added.forEach((element, index) => {
+    // updates checklist status
+    checkList[index].checked = store[index].completed;
+    store[index].index = index;
+    localStorage.setItem('todo', JSON.stringify(store));
+    const linethrough = (index) => {
+      if (checkList[index].checked) {
+        addedSection[index].style.textDecoration = 'line-through';
+      } else {
+        addedSection[index].style.textDecoration = '';
+      }
+    };
+    linethrough(index);
+
+    // window.addEventListener('DOMContentLoaded',()=>addMore(element))
```

Comment:
Please do not use unused code as comments, as it affects the cleanness of the code.